server/src/main/java/org/elasticsearch/cluster/coordination/CoordinationState.java

```diff
@@ -556,27 +556,18 @@ public interface PersistedState extends Closeable {
          * marked as committed.
          */
         default void markLastAcceptedStateAsCommitted() {
-            final ClusterState lastAcceptedState = getLastAcceptedState();
```

Comment:
I think the code is cleaner, but I failed to spot the important difference, is it just the logging you wanted out of this?

Reply:
Thanks. In that case, let us revert this and move this to a follow-up, where we can also assume that assertions are held (i.e., use `true` instead of `hasClusterUuid` later in the code).